server/package.json

```diff
@@ -6,7 +6,7 @@
   "main": "./build/index.js",
   "scripts": {
     "start": "node -r tsconfig-paths/register ./build/index.js",
-    "build": "tsc -b",
+    "build": "tsc -p tsconfig.build.json",
```

Comment:
I assume we don't have to invoke this anymore since we don't actually need the `references` bit?

Reply:
yep so we're not using TS project references anymore. but interestingly in VSCode if you remove the `references` bit from `tsconfig`, clicking into a `~shared` import brings you to the compiled `.d.ts` file instead of the `.ts` file, which is annoying DX, which is why I haven't removed it. I recognise that's not the best reason to keep it around though, so am okay to remove in a separate PR as well.